AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -316,13 +316,58 @@ def depth_of_field(property: str = 'name') -> str:
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
           - 'Camera Entity' an EditorEntity.id reference to the Camera component required for this effect.
             Must be a different entity than the one which hosts Depth of Field component.\n
+          - 'CameraEntityId Override' Override enable for CameraEntityId, bool.
+          - 'Enabled Override' Override enable for Enabled, bool.
+          - 'QualityLevel Override' Override enable for QualityLevel, bool.
+          - 'ApertureF Override' Override enable for ApertureF, range 0.0-1.0.
+          - 'FocusDistance Override' Override enable for Focus Distance, range 0.0-1.0.
+          - 'EnableAutoFocus Override' Override enable for EnableAutoFocus, bool.
+          - 'AutoFocusScreenPosition Override' Override enable for AutoFocusScreenPosition, range 0.0-1.0.
+          - 'AutoFocusSensitivity Override' Override enable for AutoFocusSensitivity, range 0.0-1.0.
+          - 'AutoFocusSpeed Override' Override enable for AutoFocusSpeed, range 0.0-1.0.
+          - 'AutoFocusDelay Override' Override enable for AutoFocusDelay, range 0.0-1.0.
+          - 'EnableDebugColoring Override' Override enable for EnableDebugColoring, bool.
+          - 'Enable Depth of Field' Enables ot disables depth of field, bool.
```

Comment:
nit: bool actions "toggle for disabling depth of field (bool default True)" not sure what the default is here this might flip to enabling default False

Reply:
Fixed! default is false